[2022-01-29 01:02:46,633] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task scheduled__2022-01-28T00:00:00+00:00 [queued]>
[2022-01-29 01:02:46,670] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task scheduled__2022-01-28T00:00:00+00:00 [queued]>
[2022-01-29 01:02:46,671] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-29 01:02:46,671] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-01-29 01:02:46,671] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-29 01:02:46,721] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2022-01-28 00:00:00+00:00
[2022-01-29 01:02:46,729] {standard_task_runner.py:52} INFO - Started process 153 to run task
[2022-01-29 01:02:46,734] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag', 'local_to_gcs_task', 'scheduled__2022-01-28T00:00:00+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmpxm64ak7k', '--error-file', '/tmp/tmputqpxso9']
[2022-01-29 01:02:46,735] {standard_task_runner.py:77} INFO - Job 7: Subtask local_to_gcs_task
[2022-01-29 01:02:46,859] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task scheduled__2022-01-28T00:00:00+00:00 [running]> on host d33a9abab046
[2022-01-29 01:02:47,032] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2022-01-28T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-28T00:00:00+00:00
[2022-01-29 01:04:25,410] {python.py:175} INFO - Done. Returned value was: None
[2022-01-29 01:04:25,461] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag, task_id=local_to_gcs_task, execution_date=20220128T000000, start_date=20220129T010246, end_date=20220129T010425
[2022-01-29 01:04:25,588] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-29 01:04:25,678] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
